### ISSUE TITLE
Tasks pane in weekly view

### ISSUE THREAD

**Issue by mauromaniforti** [fullcalendar/fullcalendar#3528]
### Feature Requests
It would be handy to have in the lower part of the weekly view of an area in which to display the corresponding day of the corresponding column task with expiration date.
This would have the same functionality present in the outlook calendar and allow you to view simultaneous
ly appointments (with dates of beginning and end) and the tasks (with the expiration date)

![image](https://cloud.githubusercontent.com/assets/5570725/22972649/ddff44aa-f37a-11e6-91bd-13d64ee37b70.png)

Thanks

**Comment by espen** [comment]
Isn't that the same as what all day events do? https://fullcalendar.io/docs/agenda/allDaySlot/

**Comment by mauromaniforti** [comment]
No, isn't, at least if you don't want to get confused between "all day appointments" and "tasks".  If you look at MS Outlook you can find 3 pane: top (all days appointments), main (Appoitnments) and bottom (expiring tasks)

**Comment by espen** [comment]
Well that's a matter of definition, it should be perfectly fine to include tasks as all day events. You can also give these events a class for different CSS styling.

FullCalendar doesn't support the concept of tasks (or todos as they are called in iCalendar) at all (though it can be used for it as mentioned). It doesn't have support for due date, duration or completeness. I'm not sure what would be required to support tasks and how exactly they should be different than events. I think also it's not wise to replicate Outlook functionality blatantly.

Perhaps the issue should be renamed to just "Tasks" as FC today has no concept of tasks?

**Comment by mauromaniforti** [comment]
Yes, it colud be a matter of definition, but tasks are no very different from events.
They have an end date and not a start one. 
In my personal opinion having a bottom pane to differentiate between daily events from "Task events" should be more clear for the users.
Sure I can use the top daily pane and differentiate the two kind of events using different colors, but in my use case I use FC to build a "Shared calendar" showing events from many users and the color is just used to differentiate the "User event" based on association user/color.
Perhaps another solution may be adding an icon on the event to differentiate (if I remember correctly there is just another issue open about that)

**Comment by espen** [comment]
If you just want to add an icon to an event you can just give the events that are task [another CSS class](https://fullcalendar.io/docs/event_data/Event_Object/).

There are not many differences between events and todos but they would have to be treated differently. They can have a starte date but unlike events todos have completed, percentage, due/duration (well actually same as end date): http://www.kanzaki.com/docs/ical/vtodo.html

btw it's also possible to implement a [custom view](https://fullcalendar.io/docs/views/Custom_Views/)

**Comment by AlexandreBonneau** [comment]
Displaying VTODO element in the agenda view would be great (where the task due-date would be used to know where to display them).
That's how Korganizer works for instance, and from a user experience perspective, it's great to know which tasks to do when in a quick glance.

**Comment by AlexandreBonneau** [comment]
In the latest Nextcloud version, you can see lines at the correct due date place for your VTODO elements, so that's a start :)
Also, VTODO created in another client (ie Korganizer) are correctly synchronized between devices through Nextcloud and fullcalendar.
